scripts/install_docker.sh

```diff
@@ -224,6 +224,7 @@ do_install() {
 			$sh_c "apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D"
 			$sh_c "mkdir -p /etc/apt/sources.list.d"
 			$sh_c "echo deb https://apt.dockerproject.org/repo ${lsb_dist}-${dist_version} ${repo} > /etc/apt/sources.list.d/docker.list"
+            $sh_c "sed -i s/xenial/wily/ /etc/apt/sources.list.d/docker.list"
```

Comment:
This wont replace all instances of the series, the sed is scoped only to xenial?